IFC4x3/Properties/r/RelativePosition_06BnzJB6b9zhMKCUAtPkfJ/DocProperty.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RelativePosition_06BnzJB6b9zhMKCUAtPkfJ" Name="RelativePosition" UniqueId="062f1f53-2c69-49f6-b594-31e2b766ea53" PropertyType="p_enumeratedvalue">
```

Comment:
Since definitions are global, maybe name="WeldedJointRelativePosition"?

Reply:
In the latest commit, it is updated to JointRelativePosition